src/common/validation/currency-schema.ts

```diff
@@ -8,7 +8,7 @@ function curencyAmountSchema() {
 
 export function stxAmountSchema(errorMsg: string) {
   return curencyAmountSchema()
-    .required('Enter an amount of STX')
+    .required('Enter a STX amount')
```

Comment:
Is this really better phrasing? cc/ @markmhx

Reply:
It must be the subpar American English you speak (LOL, jk). I dunno, maybe with the shorter version `Stacks` vs STX might be better, just kinda seems abrupt.

Reply:
https://english.stackexchange.com/a/9442 suggests we don't even use amount at all

Reply:
I'll switch it back bc already a lot of changes here!